src/main.cpp

```diff
@@ -55,17 +58,25 @@ int main(int argc, char* argv[])
 
   auto max_and_mean = MIRCO::ComputeMaxAndMean(topology);
 
-  // Initialise Pressure
-  double pressure = 0.0;
+  // Number of contact points
+  int nf = 0;

```

Comment:
`nf` is not a great name for a variable. Why not `num_contact_points`? Then, one does not need a comment to explain it.